tap_mysql/sync_strategies/binlog.py

```diff
@@ -549,7 +593,10 @@ def sync_binlog_stream(mysql_conn, config, binlog_streams, state) -> None:
 
         LOGGER.info("Starting binlog replication with log_file=%s, log_pos=%s", log_file, log_pos)
 
-        _run_binlog_sync(mysql_conn, reader, binlog_streams_map, state, config)
+        end_log_file, end_log_pos = fetch_current_log_file_and_pos(mysql_conn)
+        LOGGER.info('Current Master binlog file and pos: %s %s', end_log_file, end_log_pos)
```

Comment:
is this to move `fetch_current_log_file_and_pos` out of `_run_binlog_sync` and the functionality remains?

Reply:
Yes